### DISCUSSION TITLE
Workspaces Icons not consistent with images shown for themes.

### DISCUSSION THREAD

**Discussion by shaneosu** [amnweb/yasb#27] [Q&A] [answered]
Hi! I am new to YASB and status bars in general and was wondering if it was possible to have my komorebi workspace icons appear as they do in the images provided for the default theme of YASB? Currently both themes provided in the yasb-themes repo show the same icons, which are consistent with the acrylic theme. I've provided pictures of the 2 themes below. Any help would be greatly appreciated.

Acrylic
![image](https://github.com/user-attachments/assets/abf1ec44-2042-40be-affb-11c8a174e63f)

Default
![image2](https://github.com/user-attachments/assets/39253ee7-6b87-4832-9f43-e78727ee1d6f)

**Comment by amnweb** [comment]
You can change icons in config file, widget `komorebi_workspaces` and this lines:

```
label_workspace_btn: "\udb81\udc3d"
label_workspace_active_btn: "\udb81\udc3e"
```

**Reply by shaneosu** [reply]
Thanks for the quick reply. I was able to find the line to change icons while configuring the bar was just unsure of what specific icons were used in that image. Could you point me in the right direction?

**Reply by amnweb** [reply]
You can get the config file from these themes and check there which icons are used.

**Reply by shaneosu** [reply]
That was my first thought as well but when downloaded the theme is different from what is shown in the image, that is very my confusion comes from. This is also the case with the icons used for the Aura theme in the repository.

This is the image shown for the Default theme
![image](https://github.com/user-attachments/assets/bff4d843-fbf9-4697-a4b8-109611863ad3)

This is what the theme actually looks like when in use
![Screenshot 2024-08-29 221121](https://github.com/user-attachments/assets/d9bfb598-b1c1-45e8-bf7b-bcc9f5183baa)

**Reply by amnweb** [reply]
Oh, this image needs to be updated. I can't remember which icon that is, but you can download https://github.com/character-map-uwp/Character-Map-UWP which is great software, so you can find icons that you like and copy the UTF-16 icon in config.

![Screenshot 2024-08-30 043151](https://github.com/user-attachments/assets/6f634441-056d-43e5-b147-4dcfb60da5e2)

**Reply by amnweb** [reply, marked as answer]
BTW, based on the commit history, it looks like this isn't an icon this is only CSS.

try something like this

```
.komorebi-workspaces .ws-btn {
    border-radius: 5px;
    font-size: 20px;
    width: 10px;
    height: 10px;
    padding: 0 2px;
    color:  rgba(218, 231, 255, 0.3);
    background-color: rgba(0, 0, 0, 0);
}
.komorebi-workspaces .ws-btn:hover {
    color:  rgba(218, 231, 255, 0.5);
}
.komorebi-workspaces .ws-btn.populated {
    color:  rgba(218, 231, 255, 0.7);
}
.komorebi-workspaces .ws-btn.active {
    background-color: rgba(218, 231, 255, 0.8);
    color:  rgba(0,0,0,0);
    width: 28px;
}
```

But also in config remove icons

```
label_workspace_btn: ""
label_workspace_active_btn: ""
```

**Reply by shaneosu** [reply]
Thank you! Great work on the bar btw. I am loving it.

**Reply by shaneosu** [reply]
to get the desired effect i still needed to set an icon for the ws button for inactive workspaces but this worked great
